fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -1292,6 +1292,17 @@ countries:
                     colormap: precip
                     lower_is_worse: no
                     format: bad
+                bad-years-v3:
+                    label: Drought humanitarian ranking v3
+                    description: Drought ranking of expert committee humanitarian data review (v3)
+                    path: ethiopia/southern-oromia/bad-years-v3-ond.zarr
```

Comment:
The file for the previous version is in the `ethiopia` subdirectory and has `southern-oromia` in its filename. Here you've made a subdirectory called `southern-oromia` instead of having `southern-oromia` in the filename. Please be consistent.

Reply:
e.g. see line 1288.